Node-DC-EIS-client/runspec.py

```diff
@@ -42,7 +41,7 @@
 from itertools import izip
 from threading import Thread,Timer
 from multiprocessing import Process   
-from multiprocessing import Pool
+from multiprocessing import Queue
```

Comment:
@egalli Why use of "Queue"? How or Where does it help?

Reply:
Pool was not being used. So, I removed it.

Queue is a thread and process safe container used for transferring data between the main thread and the data processing process. The main threads tells the process process that a temporary interval log file is ready for processing (instead of sleeping for an interval of time).